components/script/webdriver_handlers.rs

```diff
@@ -1814,15 +1848,15 @@ pub(crate) fn handle_element_click(
     reply
         .send(
             // Step 3
-            get_known_element(documents, pipeline, element_id).and_then(|element| {
+            get_known_element(documents, pipeline, element_id).and_then(|ref element| {
```

Comment:
```suggestion
            get_known_element(documents, pipeline, element_id).and_then(|element| {
```
Pls revert the change. We want to make things explicit at the point of using `&element`. 
That's why we barely see such usage in closure.